services/QuillLMS/client/app/bundles/Comprehension/actions/session.ts

```diff
@@ -17,12 +17,68 @@ interface GetFeedbackArguments {
   callback: Function
 }
 
+interface FetchActiveActivitySessionArguments {
+  sessionID: string,
+  activityUID: string,
+  callback: Function
+}
+
+interface SaveActiveActivitySessionArguments {
+  sessionID: string,
+  submittedResponses: { [key: string]: FeedbackObject[] }|{},
+  activeStep: number,
+  completedSteps: number[],
+  callback: Function
+}
+
 export const setSessionId = (sessionID: string) => {
   return (dispatch: Function) => {
     dispatch({ type: ActionTypes.SET_ACTIVITY_SESSION_ID, sessionID });
   }
 }
 
+export const fetchActiveActivitySession = (args: FetchActiveActivitySessionArguments) => {
+  const { sessionID, activityUID, callback, } = args
```

Comment:
this is nbd but you can do `export const fetchActiveActivitySession = ({ sessionID, activityUID, callback, }: FetchActiveActivitySessionArguments) => {` and then you don't need the following line.